src/test/java/org/junit/tests/experimental/AssumptionTest.java

```diff
@@ -94,10 +99,15 @@ public void assumeThatPassesOnStrings() {
         assertCompletesNormally();
     }
 
-    @Test(expected = AssumptionViolatedException.class)
+    @Test
     public void assumeNotNullThrowsException() {
-        Object[] objects = {1, 2, null};
-        assumeNotNull(objects);
+        try {
+            Object[] objects = {1, 2, null};
```

Comment:
Done but I will probably revert 6413e99bb6255033199709c7ed15713610399e0d as I've found a better fix in #1291